src/test/ui/lifetime-errors/ex3-both-anon-regions-earlybound-regions.rs

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
missing license and newlines in the code

Reply:
also, shouldn't this test go with the other PR?

Reply:
Yeah. Was a mistake